lib/core/stores/remote.ts

```diff
@@ -385,17 +385,23 @@ export class RemoteScheduleStore implements IScheduleStore {
 }
 
 export class RemoteLockStore implements ILockStore {
-  private lockExpiry: number = 60000;
-  private heartbeatDelay: number = this.lockExpiry / 4;
+  private lockTimeout: number = 60000;
+  private heartbeatDelay: number = this.lockTimeout / 4;
   private hearbeatInterval: number | null = null;
 
   constructor(
     private url: string,
     private pid: string,
     private logger: ILogger = new Logger(),
-  ) {}
+    private lockExpiry?: number,
```

Comment:
Here's a convenience so that we don't have to conditionally set the class member

```suggestion
    private lockTimeout: number = 60000,
```